src/server/frontEnd/src/utils/sockets/ws-routing.js

```diff
@@ -1,5 +1,5 @@
 const io = require('socket.io-client')
-const IO_CONNECTION_URL = 'localhost:5000/room' /* Take this from config file in the future */
+const IO_CONNECTION_URL = `${document.location.host}/room`
```

Comment:
We shouldn't create new room every time when during user logging. Server should create room only when one user want to talk with someone else.